geologist/child-theme.json

```diff
@@ -333,7 +333,7 @@
 					"link": "var(--wp--custom--color--primary)"
 				},
 				"typography": {
-					"fontSize": "18px",
+					"fontSize": "var(--wp--preset--font-size--normal)",
```

Comment:
Should this one be small?

Reply:
No, same as the above comment, for Site Title - same as Quadrat https://github.com/Automattic/themes/pull/4683#discussion_r714278696